docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -64,3 +64,9 @@ Testing
   test/tools/router_check/test/config/... . ::
 
     bazel test //test/tools/router_check/...
+  
+Test schema
+  The tool deals with JSON objects internally. A recent feature moves all internal schemas in the tool to proto3. 
+  This feature can be enabled by an extra optional parameter ``--useproto``.
+  It introduces a breaking change in the test based on `schema <https://github.com/envoyproxy/envoy/blob/master/test/tools/router_check/validation.proto>`_.
```

Comment:
What do you mean a breaking change?

Reply:
@mattklein123 the current tests are a headless array of json. The proto3 message does not accept it and needs root level named elements.

Reply:
Please make new tests to cover the new code paths.

Reply:
@mattklein123 Yes, I am working on this today.